File: leetcode/58-length-of-last-word/length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        new_string = s[::-1]
        cnt = 0

        print(new_string)
        print(s[::-1])
        print(new_string[0])
        for i in range(len(new_string)):
            if new_string[i] != ' ':
                cnt += 1
            elif cnt > 0:
                return cnt 
        return cnt
        

File: 58-length-of-last-word/length-of-last-word.rs
impl Solution {
    pub fn length_of_last_word(s: String) -> i32 {
        let words: Vec<&str> = s.trim().split_whitespace().collect();
        match words.last() {
            Some(&last_word) => last_word.len() as i32,
            None => 0,
        }
    }
}